docs/configuration.adoc

```diff
@@ -34,16 +36,16 @@ are marked as important ones.
 
 This strategy can be configured in several ways through Java system properties:
 
-* `scm.range.head` and `scm.range.tail` are used to set the range of commits you want to inspect for new tests.
+* `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="COMMIT"]` and `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="LAST_COMMITS"]` are used to set the range of commits you want to inspect for new tests.
```

Comment:
the second property seems to be wrong: `ScmRunnerProperties#LAST_COMMITS = "scm.last.changes";`
there should used be `PREVIOUS_COMMIT` variable

Reply:
https://github.com/arquillian/smart-testing/blob/master/core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java

Reply:
Fixed.

Reply:
Here is another problem with the generation process - the first variable is property as: `scm.range.tail`, so the extension/generator matches the first variable that contains the given string `"COMMIT"` which is in this case: `PREVIOUS_COMMIT = "scm.range.tail"`

Reply:
YEs same problem as in other places, that when regexp thing